sql/moz-fx-data-shared-prod/org_mozilla_mozregression/mozregression_aggregates_wchan1/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.org_mozilla_mozregression.mozregression_aggregates_wchan1`
+AS
+SELECT
+  *
+FROM
+  `moz-fx-data-shared-prod.org_mozilla_mozregression_derived.mozregression_aggregates_wchan1_v1`
```

Comment:
Is there a specific reason why your handle is in both the table name and the view name? Is this short term only or still work in progress?

Reply:
The example used in the instructions already existed so I added a suffix to differentiate.